### PR TITLE
[#173994448] Implement Helper Contact Info

### PR DESCRIPTION
**Pull request by britneyepps**
- Created Helper Contact Info screen
- Added Helper Contact Info screen to helper workflow
- Map information to PDF & XML

### CODE REVIEW

File: src/main/resources/OnlineApplication.xsd [hunk 1]
```diff
@@ -2684,6 +2684,27 @@
...
+					</xsd:annotation>
+				</xsd:element>
+				<xsd:element name="Representative" type="model:RepresentativeType" minOccurs="0">
+					<xsd:annotation>
+						<xsd:documentation>F# 82 - 96</xsd:documentation>
+					</xsd:annotation>
+				</xsd:element>
+			</xsd:sequence>
+		</xsd:complexType>
+	</xsd:element>
```
Comment (luigi) [inline src/main/resources/OnlineApplication.xsd:2707]: This didn't need to be added. It's already on line 744.

https://github.com/codeforamerica/shiba/blob/0a1bd1668d3f2643cebc1029e091bb28cb758b15/src/main/resources/OnlineApplication.xsd#L744-L764

**Review by luigi — CHANGES_REQUESTED** [review]
Other than the deletion I flagged, everything looks good. :+1:

File: src/main/resources/OnlineApplication.xsd (hunk 1 above)
Comment (britneyepps) [reply src/main/resources/OnlineApplication.xsd]: Thanks for that catch!

**Comment by bencalegari** [comment]
@britneyepps It looks like the initial rebase you did was right since there aren't any merge commits, but in the push when you added [911bc154](https://github.com/codeforamerica/shiba/pull/47/commits/11bc154b953c242ac22908c4a5571b457e17aa4a) it did created a [merge commit], indicating you may have ran `git merge master` rather than `git rebase master`. The latter wouldn't hav e made a merge commit I think.

**Pull request merged**